.github/workflows/check-versions.yaml

```diff
@@ -32,6 +32,25 @@ jobs:
           ./.github/scripts/update-versions.sh
           git diff
 
+      - name: Setup go
+        uses: actions/setup-go@v4
```

Comment:
Can we lock the action on the patch release.

```suggestion
        uses: actions/setup-go@v4.0.0
```

Reply:
good idea.